packages/assistant-stream/src/core/serialization/DataStream.ts

```
@@ -59,6 +59,10 @@ export class DataStreamEncoder {
               );
               break;
 
+            case "error":
+              controller.enqueue("3:" + JSON.stringify(chunk.error) + "\n");
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)